web/satellite/src/components/header/projectsDropdown/ProjectDropdown.vue

```diff
@@ -22,6 +22,9 @@
             >
                 <p class="project-dropdown__wrap__choice__unselected">{{ project.name }}</p>
             </div>
+						<div @click="toggleSelection" @click.prevent.stop="closeDropdown" class="project-dropdown__wrap__create-project__wrapper">
```

Comment:
this change is here again. It should be made in a separate PR

Reply:
sorry didn't mean for this to get in here